### PR TITLE
Hosted Private Cloud : Backup with Veeam on VCD guides

### PR DESCRIPTION
**Pull request by pg-ovh**
VEEAM translations are not yet delivered in VCD UI for now.

### CODE REVIEW

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+- Les informations sur les réseaux **vApp** et les réseaux d'organisation auxquels le vApp est connecté.
+- Les options de démarrage des **VM** (VM Startup options).
+- Les informations utilisateurs.
+- Le bail (Lease).
+- Le quota.
+- Les Modèles de stockages (Templates).
+
+Les métadonnées vApp/VM sont stockées avec le contenu de la machine virtuelle. La capture des métadonnées vApp/VM est extrêmement importante pour la restauration : sans elle, vous ne serez pas en mesure de restaurer les vApp et les VM vers VMware Cloud Director.
+
+<!--- proofread stops below this point --->
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: This part (from line 84 to line 97) can't be proofread by our team.
The information is confusing (the different options/repository/storage offers lack descriptions) and the tone (especially the use of "je") is not appropriate.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+### Comment sauvegarder une machine virtuelle avec Veeam Backup ?
+
+> [!primary]
+>
+> Aucun agent n'est nécessaire au fonctionnement des sauvegardes avec Veeam Backup Data Protection depuis une machine virtuelle ou une vApp.
+>
+
+> [!warning]
+> Pour pouvoir ajouter un job depuis la console VCD sur une VM, un JOB doit être pre ????
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: Incomplete sentence.

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]
Please consider rewriting the part between line 84 and line 97

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+updated: 2024-05-16
+---
+
+## Objectif
+
+**Nous allons voir dans ce guide les solutions de sauvegarde et de restauration de Veeam pour VMware Cloud Director.**
+
+## Prérequis
+
+- Posséder une offre [Hosted Private cloud](https://www.ovhcloud.com/fr/enterprise/products/hosted-private-cloud/){.external}.
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: we need to remove this

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+- Un compte vCloud Director administrateur avec une Organisation VCD.
+- Un utilisateur avec le rôle Administrateur de l'organisation pour vous connecter au portail en libre-service Veeam Data Protection (le nouvel utilisateur admin d'un datacenter virtuel a le rôle par défaut).
+- Avoir lu les guides VCD : [Les concepts fondamentaux](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-get-concepts) / [Comment se connecter à son organisation](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-logging) / [Comment utiliser l'interface utilisateur](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started)
+
+## En pratique
+
+Veeam Backup & Replication accompagne VCD.
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: Veeam Data Platform

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+<!--- proofread stops below this point --->
+
+##### Les politiques de sauvegardes
+
+##### Les depots (repository)
+
+Par défaut, vous avez **les dépots** suivant :
+
+Avec l'offre que vous avez souscrite vous disposer de 3 depots avec 2 Jobs configurés par défaut avec un quota de 100GB :
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: pas de Job configurés finalement.  
Pas de quota non plus.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+1. **Bronze Repository** : STD Stockage Objet
+2. **Silver Repository** : STD Stockage Objet + Sauvegarde hors Site (Offsite)
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: Copy de sauvegarde hors site ( offsite ).

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+3. **Gold Repository** : Haute performance stockage object + Sauvegarde hors site (offsite) + 14 points d'immutabilité (14 points de restaurations sur 2 sites différents)
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: Copie de sauvegarde hors site (offsite)

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+Les référentiels sont tous configurés avec l'Object Storage Standard. Seul le produit de disponibilité générale répond aux spécifications de référentiel indiquées.
```
Comment (franckcosta) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: to remove

**Review by franckcosta — COMMENTED** [review]
can you check @pg-ovh  ?

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md (hunk 7 above)
Comment (pg-ovh) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-backup/guide.fr-fr.md]: Comme ça @franckcosta ? :
Par défaut, vous avez **les dépots** suivant :
1. **Bronze Repository** : STD Stockage Objet
2. **Silver Repository** : STD Stockage Objet + Copie de sauvegarde hors site ( offsite ).
3. **Gold Repository** : Haute performance stockage object + Copie de sauvegarde hors site (offsite)

**Comment by pg-ovh** [comment]
Hello @OVH-Team-Guides I have added your review. I have not included a table of content since you do not recommend it in general, but I would consider it in this guide. Edition done.

**Review by pg-ovh — COMMENTED** [review]
Ok for me @Y0Coss @OVH-Team-Guides

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @pg-ovh

**Pull request merged**